hca-api-stub/voa-rest.js

```diff
@@ -1,21 +1,83 @@
+const fs = require('fs');
 const loopback = require('loopback');
 const path = require('path');
 
+// TODO: node.js did not like `securityArtifacts` being declared as `let`. Why?
```

Comment:
Sigh. cause we need to enable strict mode or something weird. TODO is good enough for now.